### ISSUE TITLE
A member is not displayed in the list

### ISSUE THREAD

**Issue by netfreak98** [inex/IXP-Manager#497]
A new IXP member does not appear on "https://manager.example.com/switch/configuration" if IPv4 is not enabled on its port.

**Comment by netfreak98** [comment]
We're using manager version 1.4.9

**Comment by barryo** [comment]
Hi @netfreak98 - I tried (hard) to reproduce this but I could not. I have tidied the code but it should not affect the bug you were reported.

I'm going to close now but if you can reliably reproduce this, could you:

1. provide exact steps
2. describe your virtual interface in detail please (number of physical interfaces and state of each, vlan interface details)

Thanks, Barry

**Comment by netfreak98** [comment]
Hi Barry,

I just made you some screenshots.
![1](https://user-images.githubusercontent.com/31798580/51233232-a9a60100-1969-11e9-923b-b357732dcd2d.png)
![2](https://user-images.githubusercontent.com/31798580/51233234-aa3e9780-1969-11e9-9265-1c8b752fc7cd.png)
![3](https://user-images.githubusercontent.com/31798580/51233236-aa3e9780-1969-11e9-8833-bccb48013ebe.png)
![4](https://user-images.githubusercontent.com/31798580/51233233-a9a60100-1969-11e9-9136-987d870c011a.png)

**Comment by netfreak98** [comment]
We are using version V4.9.0

**Comment by barryo** [comment]
@netfreak98 - could you update to the current v4 code base? You just need to:

```sh
cd /srv/ixpmanager    # or as appropriate
git fetch --all
git checkout release-v4
```

Nothing else required. `release-v4` will track bug and security fixes for v4 that would eventually be a v4.9.x release.

Can you confirm the issue still exists there?

**Comment by netfreak98** [comment]
As a company that uses the manager in a productive environment, we are not happy to switch to an unstable branch. Is there an alternative solution here?

**Comment by barryo** [comment]
Hi @netfreak98 - you are advised to move to v4.9.1 as v4.9.0 contained a couple breaking regressions - see: https://github.com/inex/IXP-Manager/releases/tag/v4.9.1

The `release-v4` is the stable branch - it only receives bug fixes and security fixes for the v4.9 (which is the last minor release that v4 will get).

You can see there are only a handful of commits there: https://github.com/inex/IXP-Manager/commits/release-v4

If you want to wait until v4.9.2 that's fine, we can put this on ice. But I still cannot reproduce your issue so will need confirmation it still exists in 4.9.2 when released / release-v4 as it stands now.

Thanks, 
Barry

**Comment by netfreak98** [comment]
great, the problem is now resolved :)
